File: libvoikko/java/src/main/java/org/puimula/libvoikko/SentenceStartType.java
package org.puimula.libvoikko;

/** Sentence start types */
public enum SentenceStartType {
    /** End of text reached or error. */
    NONE,
    /** This is not a start of a new sentence. */
    NO_START,
    /** This is a probable start of a new sentence. */
    PROBABLE,
    /** This may be a start of a new sentence. */
    POSSIBLE;
}

File: libvoikko/cs/libvoikko/SentenceStartType.cs
using System;
namespace libvoikko
{
	public enum SentenceStartType
	{
		/// <summary>
		/// End of text reached or error.
		/// </summary>
		NONE,
		/// <summary>
		/// This is not a start of a new sentence.
		/// </summary>
		NO_START,
		/// <summary>
		/// This is a probable start of a new sentence.
		/// </summary>
		PROBABLE,
		/// <summary>
		/// This may be a start of a new sentence.
		/// </summary>
		POSSIBLE
	}
}
